system/templates/rolls/roll-graphic.hbs

```diff
@@ -0,0 +1,54 @@
+<div class="dice-roll ironsworn-roll">
+
+  <div class="dice-formula">
+    <p>
+      {{!-- The action die or progress score --}}
+      <span class="action roll d6 {{actionMinMax}}
+        {{#if actionCanceledByNegativeMomentum}}canceled{{/if}}
+      " title="{{computedActionDie.source}}">
+        {{#if computedActionDie.value}}
+        {{computedActionDie.value}}
+        {{else}}
+        &nbsp;
+        {{/if}}
+      </span>
+
+      {{#each adds}}
+      &plus;
+      <span title="{{source}}">{{value}}</span>
+      {{/each}}
+
+      {{#if actionTotal}}
+      =
+      <strong>
+        {{#if actionTotalCapped}}
+        <abbr title="{{localize 'IRONSWORN.CappedAt10'}}">{{actionTotal}}</abbr>
```

Comment:
`<abbr>` is specifically for abbreviations and acronyms. there isn't a clear-cut alternative, but a `<u>` (hey, the annotation's _unarticulated_ + nontextual before you hover it, right?) or a generic `<span>` with a class describing the sort of annotation it is.

Reply:
Yeah, it's not exactly the right thing, but it came with the visual I was looking for in a v1.
![CleanShot 2022-08-19 at 16 41 44](https://user-images.githubusercontent.com/39902/185719849-6cd8c557-2344-4631-b69e-476d4d2d60f3.jpg)

Let's come back to this when we do the work for #366.